src/ast-utils/docs.ts

```diff
@@ -130,8 +130,10 @@ function copyProperties(
       if (fromEvent.__fromBehavior) {
         return;
       }
-      var newEvent = Object.create(fromEvent);
-      newEvent.__fromBehavior = from.is;
+      var newEvent = {__fromBehavior: from.is};
+      Object.keys(fromEvent).forEach(function(eventField){
```

Comment:
It can't, you could do:

```
let x = Object.assign;
// things that can't be statically determined
x(a, b);
```

It needs to be added to `Object`